libs/SalesforceReact/SalesforceReactTests/ReactNetTests.m

```diff
@@ -22,9 +22,36 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <UIKit/UIKit.h>
+#import "ReactTestCase.h"
 
-#import "SFPluginTestSuite.h"
+@interface ReactNetTests : ReactTestCase
 
+@end
+
+@implementation ReactNetTests
+
+- (void)setUp {
+    self.jsSuitePath = @"js/net.test";
```

Comment:
Test file on the objective c is very simple.
Simply specify the javascript test file in `setUp()` (without the `.js`).
For each test have a `RCT_TEST(xxx)` if the test is called `testxxx`.